.github/workflows/windows-latest-mingw.yml

```diff
@@ -60,7 +62,7 @@ jobs:
 
     - name: Build
       # Build program
-      run: cmake --build ${GITHUB_WORKSPACE}/build
+      run: cmake --build ${GITHUB_WORKSPACE}/build --config ${{env.BUILD_TYPE}}
```

Comment:
MinGW doesn't use multi-config generators. Setting `--config` here isn't necessary. Instead the value of `CMAKE_BUILD_TYPE` in the configure step is decisive (like for most other platforms). 
(The `--config` flag in the build step might be a co-op.)